platform-tests/src/acceptance/rds_broker_test.go

```diff
@@ -105,6 +106,93 @@ var _ = Describe("RDS broker", func() {
 	})
 })
 
+var _ = Describe("RDS broker - MySQL", func() {
+	const (
+		serviceName              = "mysql"
+		testPlanName             = "Free"
+		testPlanNameWithSnapshot = "S-dedicated-5.7"
+		region                   = "eu-west-1"
```

Comment:
Leftover from postgres tests before the [recent merge](https://github.com/alphagov/paas-cf/commit/431ad38ffdd67e1f3b3296e0eecf998b2edf188b#diff-ebbb4adbeb7f9ea59f6a7015a3a3afbaL30).

Fixing. Thanks.